xgboost-operator.yaml

```diff
@@ -0,0 +1,201 @@
+#--------------------------------------------------
+#- Single configuration deployment YAML for xgboost-Operator
+#- Include:
+#      CRD deployment
+#      Namespace deployment
+#      RBAC deployment
+#      xgboost-operator container deployment set     
+# --------------------------------------------------
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: xgboostjobs.xgboostjob.kubeflow.org
+spec:
+  group: xgboostjob.kubeflow.org
+  version: v1alpha1
+  names:
+    plural: xgboostjobs
+    kind: XGBoostJob
+    shortNames:
+    - xgbj
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            slotsPerWorker:
+              type: integer
+              minimum: 1
+            xgbReplicaSpecs:
+              properties:
+                Launcher:
+                  properties:
+                    replicas:
+                      type: integer
+                      minimum: 1
+                      maximum: 1
+                Worker: 
+                  properties:
+                    replicas:
+                      type: integer
+                      minimum: 1
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: xgboost-operator
```

Comment:
namespace would have a better name rather than operator name. 
for example, test-namespace or my-namespace

Reply:
sure, I will make some changes

Reply:
already changed, appreciate your further review